package.json

```diff
@@ -89,5 +90,9 @@
     "versionCompatibility": {
       "ember": "3.8 || 3.12 || 3.16 || 3.20 || >=3.24"
     }
+  },
+  "volta": {
```

Comment:
Might be useful for future Volta users, though I understand if you'd want this removed.